packages/ruleset/test/utils/root-document.js

```diff
@@ -669,13 +669,73 @@ module.exports = {
             maxLength: 1024,
           },
         },
+        example: {
+          id: 'acb123',
+          name: 'The Two Towers',
+          director: 'Peter Jackson',
+          running_time: 179,
```

Comment:
Some of these other changes are to address Spectral-core rules that flagged things about this document when I exported it to run the whole validator against it. Just house cleaning changes